plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -390,6 +390,11 @@ func (p *Plugin) admitPVCStatus(nodeName string, a admission.Attributes) error {
 
 func (p *Plugin) admitNode(nodeName string, a admission.Attributes) error {
 	requestedName := a.GetName()
+
+	if requestedName != nodeName {
+		return admission.NewForbidden(a, fmt.Errorf("node %q is not allowed to create or modify node %q", nodeName, requestedName))
```

Comment:
keep the message text the same as it was previously and revert the unit test change... I didn't notice the message had changed

Reply:
Done ;-)